code/obj/item/space_cash.dm

```diff
@@ -99,6 +99,20 @@
 		src.UpdateStackAppearance()
 		return src
 
+	attackby(obj/item/I, mob/user)
```

Comment:
This should use `stack_item` rather than attackby to account for other sources of stacking